src/commands/cm/create.ts

```
@@ -0,0 +1,355 @@
+                    console.error(`Failed to read collection JSON file: ${error instanceof Error ? error.message : error}`)
+                }
+            }
+
+        }
+        if (candyMachineConfig.config.collection) {
+            this.log(`- Collection ID: ${candyMachineConfig.config.collection}`)
+        }
+        const hasGlobalGuards = candyMachineConfig.config.guardConfig && Object.keys(candyMachineConfig.config.guardConfig).length > 0
+        const hasGroups = candyMachineConfig.config.groups && candyMachineConfig.config.groups.length > 0
+        if (hasGlobalGuards) {
+            this.log(`- Global guards: ${Object.keys(candyMachineConfig.config.guardConfig!).join(', ')}`)
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)